lib/types/application/ApplicationBuilder.js

```diff
@@ -86,7 +68,7 @@ class ApplicationBuilder extends AbstractBuilder {
 		const componentPreload = project.builder && project.builder.componentPreload;
 		if (componentPreload) {
 			this.addTask("generateComponentPreload", async () => {
```

Comment:
[minor][consistency] why is this one (and the next one) marked as async? To me, the calls don't look any different to the other calls.

Reply:
Good point. Since getTask can throw synchronously, we should always make the  functions async so that the throw becomes a promise rejection and can be handled correctly in the promise chain.